web/files/files.go

```diff
@@ -208,46 +218,61 @@ func ModificationHandler(c *gin.Context) {
 	}
 
 	patchData := container.Data
-	docType, err := vfs.ParseDocType(patchData.Type)
+
+	fileID := c.Param("file-id")
+
+	var docType vfs.DocType
+	if fileID == MetadataPath {
```

Comment:
It would be clearer to have two handler, 
```
router.GET("/metadata", ModificationMetadataHandler)
router.GET("/:file-id", ModificationContentHandler)
```